components/Comment.js

```diff
@@ -55,6 +71,17 @@ class Comment extends React.Component {
     this.state.mode === 'edit' ? this.cancelEdit() : this.edit();
   }
 
+  handleDelete = async () => {
+    try {
+      await this.props.deleteComment(this.state.comment.id);
+      await this.props.refetch();
+      this.setState({ showDeleteModal: false });
+    } catch (err) {
+      console.error(err);
+      this.setState({ showDeleteModal: false });
+    }
```

Comment:
Not a good practice to hide errors (especially because `console.error` doesn't notify us).